Cargo.toml

```diff
@@ -148,8 +148,11 @@ default-members = [
 ]
 
 [profile.release]
+opt-level = 3
 debug = true
-overflow-checks = true
+overflow-checks = false
+lto = true
+codegen-units = 1
```

Comment:
This will slow down the build time significantly I believe?

Reply:
It does. If this impacts dev tme, I can just add another profile, that we can use discriminately.

Reply:
let's make a different profile, release build is already slow enough

Reply:
all up : 8501 TPS, 3478 ms latency, 6000 ms p99 latency,no expired txns.
With 28m build time. Now gated behind `cargo build --profile performance`